static/scss/main.scss

```diff
@@ -217,48 +219,129 @@ a#skipnav {
 
 #home {
   .hero {
-    @include span-columns(8);
-    @include shift(2);
-
     text-align: center;
     color: white;
-    padding-top: 40px;
+    padding: 40px 0 60px 0;
 
     h2 {
       font-size: $base-font-size * 7;
       font-weight: 300;
     }
     h3 {
+      display: inline-block;
       font-size: $base-font-size * 2;
       font-weight: 100;
       padding-top: 15px;
+      max-width: 550px;
     }
   }
 
-  #results {
-    @include span-columns(8);
-    @include shift(2);
+  #wrapper {
+    @include clearfix();
 
-    padding: 40px 0;
+    background: #fff;
+    clear: both;
+    color: #000;
+
+    h4 {
+      font-size: 2em;
+    }
+  }
+
+  #results {
+    clear: both;
+    margin: 0 auto;
+    max-width: 840px;
+    padding: 40px 40px;
 
     .https, .dap {
-      @include span-columns(4 of 8);
-      @media (max-width: 970px) {
-        width: 100%
+      margin-bottom: 10px;
+
+      .total-value {
+        font-size: 3em;
+      }
+      .total-desc {
+        font-size: 0.8em;
+      }
+
+      .https_chart, .dap_chart {
+        margin: 0 auto;
+        width: 200px;
       }
 
-      @include display(flex);
-      @include flex-direction(column);
-      @include align-items(center);
-      @include justify-content(center);
+      .https_chart {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Selector `https_chart` should be written in lowercase with hyphens